frontend/views/containers/dashboard/GroupMembers.vue

```diff
@@ -8,8 +8,7 @@
       @click='headerButtonAction'
     )
       i.icon-plus.is-prefix
-      i18n(v-if='action === "addMember"') Add
-      i18n(v-else) New
```

Comment:
Why is this logic changed? 🤔

Reply:
@SebinSong I believe you made this change, please clarify if I'm mistaken. I believe this related to fixing the issue with groups being unable to add new members if the invite link is expired or used up and there are no members.

Reply:
please check this comment:
https://github.com/okTurtles/group-income/pull/2351#discussion_r1760569165

The `action` prop has only has `addMember` as the option and no other value. So I just removed the prop. (Looks like there used to be other options in earlier days)